src/dialogs/more-info/ha-more-info-logbook.ts

```diff
@@ -44,7 +45,11 @@ export class MoreInfoLogbook extends LitElement {
       return html``;
     }
 
-    const href = "/logbook?entity_id=" + this.entityId;
+    const href =
+      "/logbook?entity_id=" +
+      this.entityId +
+      "&start_date=" +
+      startOfYesterday().getTime();
```

Comment:
Can we not calculate this on every render, either do it just on click (but that will break normal link actions) or calc it only on the first will update (as it will not update much anyway)

Reply:
And let's use an iso string instead of time